code/modules/recycling/conveyor2.dm

```diff
@@ -55,6 +56,7 @@ GLOBAL_LIST_EMPTY(conveyors_by_id)
 	if(newid)
 		id = newid
 	update_move_direction()
+	RegisterSignal(src, COMSIG_MOVABLE_CROSSED, .Crossed) // For receiving signals from newly created items. These do not call Crossed() so it needs a signal
```

Comment:
```suggestion
	RegisterSignal(src, COMSIG_MOVABLE_CROSSED, .proc/Crossed) // For receiving signals from newly created items. These do not call Crossed() so it needs a signal
```

Reply:
Yeah it is cursed. I'm just changing the proc name to `conveyorCrossed`, which works fine